### PR TITLE
Add doc-update-roadmap.md

### PR DESCRIPTION
**Pull request by sacha-l**

### CODE REVIEW

**Review by lisa-parity — COMMENTED** [review]
some suggestions

**Comment by NukeManDan** [comment]
We should convert this to an issue and pin to this repo instead per talking with Lisa and Shawn

**Review by NukeManDan — CHANGES_REQUESTED** [review]
I can help get the issue templates updated, and update the new issue including the roadmap with links to them once ready

File: doc-update-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  -[] Hate it! 
+  Tell us more:
+
+Content question :missing: :couldn’t find: :other:
+  Link to an issue template with fields like:
+  Topic I was looking for:
+  Where I looked (or where I expected to find it):
+  Keywords I used to search for my topics:
+
+Bug report: :broken link: :code error: :content issue (wrong, unclear, out of date)
```
[the remark this answers is not in the document]
Comment (NukeManDan) [reply doc-update-roadmap.md:59]: So you want to make a monolithic issue template? if so, is there an example of best practice for this I can emulate? I personally like the landing page we have with a couple options to help categorize and auto-tag for us. I don't think that is possible with just one issue for everything (would need to manually tag them all at least)

**Comment by NukeManDan** [comment]
I will convert to issue and send a link to Jonny !

**Comment by NukeManDan** [comment]
https://github.com/substrate-developer-hub/substrate-docs/issues/1131

**Pull request closed without merging**